docs/content/en/open_source/contributing/how-to-write-a-parser.md

```diff
@@ -168,14 +168,22 @@ Good example:
 ### Do not parse CVSS by hand (vector, score or severity)
 
 Data can have `CVSS` vectors or scores. Don't write your own CVSS score algorithm.
-For parser, we rely on module `cvss`.
+For parser, we rely on module `cvss`. But we also have a helper method to validate the vector and extract the base score and severity from it.
+
+```python
+cvss_data = parse_cvss_data("CVSS:3.0/S:C/C:H/I:H/A:N/AV:P/AC:H/PR:H/UI:R/E:H/RL:O/RC:R/CR:H/IR:X/AR:X/MAC:H/MPR:X/MUI:X/MC:L/MA:X")
+if cvss_data:
+    finding.cvssv3 = cvss_data.get("vector")
+    finding.cvssv3_score = cvss_data.get("score")
+    finding.severity = cvss_data.get("severity")  # if your tool does generate severity
+```
 
-It's easy to use and will make the parser aligned with the rest of the code.
+If you need more manual processing, you can parse the `CVSS` vector directly.
 
 Example of use:
 
 ```python
-from cvss.cvss3 import CVSS3
+from dojo.utils import cvss.cvss3 import CVSS3
```

Comment:
Not sure which import should be the correct one

Reply:
addressed this